src/components/CommentForm/CommentForm.tsx

```diff
@@ -43,15 +43,48 @@ const commentTextArea = css`
   }
 `;
 
-const placeholderCommentStyles = css`
+// https://stackoverflow.com/questions/9451902/changing-an-inputs-html5-placeholder-color-with-css-does-not-work-on-chrome
+const greyPlaceholder = css`
+  ::-webkit-input-placeholder {
```

Comment:
I thought emotion would compile down our CSS using Babel to support all browsers, so we can just simply use `::placeholder`?

https://github.com/emotion-js/emotion/tree/master/packages/babel-plugin-emotion

worth double checking what CSS is used on rendering

Reply:
You're completely right! The problem is solved with the opacity trick and these prefixes aren't needed.